CHANGELOG.adoc

```diff
@@ -24,6 +24,26 @@ title: Changelog
 * The support for the embedded Elasticsearch will be dropped in the future. It is highly recommended to link:{{< relref 
 "elasticsearch.asciidoc" >}}#_dedicated_elasticsearch[setup Elasticsearch as a dedicated service].
 
+[[v0.35.0]]
+== 0.35.0 (TBD)
+
+CAUTION: Java Rest Client: The default field values for `NodeResponse` and `AbstractGenericRestResponse` has been updated. 
+
+The following values have been changed:
+
+* `AbstractGenericRestResponse#permissions`
+* `NodeResponse#tags`
+* `NodeResponse#childrenInfo`
+* `NodeResponse#container`
+* `NodeResponse#fields`
+* `NodeResponse#breadcrumb`
+
+The `NodeResponse#isContainer` method has been deprecated and was replaced by `NodeResponse#getContainer`
+
+Applications which use affected models will most likly not needed to updated since the returned response will always set the affected fields.
```

Comment:
```suggestion
Applications which use affected models will most likely not need to updated since the returned response will always set the affected fields.
```